src/Build.UnitTests/BackEnd/BuildManager_Tests.cs

```diff
@@ -4088,8 +4088,8 @@ public void GraphBuildInvalid()
 
             GraphBuildResult result = _buildManager.Build(_parameters, data);
             result.OverallResult.ShouldBe(BuildResultCode.Failure);
-            result.Exception.ShouldBeOfType<AggregateException>().InnerExceptions.Count.ShouldBe(1);
-            result.Exception.ShouldBeOfType<AggregateException>().InnerExceptions[0].ShouldBeOfType<InvalidProjectFileException>().ProjectFile.ShouldBe(project2);
+            result.Exception.ShouldBeOfType<InvalidProjectFileException>();
+            result.Exception.ShouldBeOfType<InvalidProjectFileException>().ProjectFile.ShouldBe(project2);
```

Comment:
Nit: You either don't need this line or don't need the `ShouldBeOfType` in the line below.